render/detailed_node.go

```diff
@@ -321,17 +332,17 @@ func containerOriginTable(nmd report.Node, addHostTag bool) (Table, bool) {
 	}
 	rows = append(rows, getDockerLabelRows(nmd)...)
 
-	if val, ok := nmd.Metadata[docker.MemoryUsage]; ok {
-		memory, err := strconv.ParseFloat(val, 64)
-		if err == nil {
-			memoryStr := fmt.Sprintf("%0.2f", memory/float64(mb))
-			rows = append(rows, Row{Key: "Memory Usage (MB):", ValueMajor: memoryStr, ValueMinor: ""})
-		}
-	}
 	if addHostTag {
 		rows = append([]Row{{Key: "Host", ValueMajor: report.ExtractHostID(nmd)}}, rows...)
 	}
 
+	if val, ok := nmd.Metrics[docker.MemoryUsage]; ok {
+		rows = append(rows, sparklineRow("Memory Usage", "%0.2f MB", 1024*1024, val))
+	}
+	if val, ok := nmd.Metrics[docker.CPUTotalUsage]; ok {
+		rows = append(rows, sparklineRow("CPU Usage", "%0.2f%%", 1, val))
+	}
+
```

Comment:
Stick this is a for loop with the anonymous type?  I have the feeling we'll be adding more here shortly.